src/configure/configure.ts

```diff
@@ -145,6 +147,7 @@ class Orchestrator {
                     this.inputs.pipelineConfiguration.template = shortlistedTemplates[0];
                 }
                 else {
+                    telemetryHelper.setTelemetry(TelemetryKeys.TemplateNotFound, Messages.TemplateNotFound);
```

Comment:
If I am not wrong `telemetryHelper.setTelemetry` will add this telemetry point to the command's log. That is good for any customer intelligence kind of data.
For exceptions, should we use `telemetryHelper.logError`

Reply:
The next line which throws the error will automatically add this error to the command's log (as `error` property) as well so no need add one extra property there.
If you want to log this error so that you can find all such template not found errors, you should use `telemetryHelper.logError`

Reply:
updated